frontend/pages/hosts/ManageHostsPage/components/DeleteSecretModal/DeleteSecretModal.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import Modal from "components/Modal";
+import Button from "components/buttons/Button";
+import { ITeam } from "interfaces/team";
+import { IEnrollSecret } from "interfaces/enroll_secret";
+
+interface IDeleteSecretModal {
+  selectedTeam: number;
+  teams: ITeam[];
+  onDeleteSecret: () => void;
+  toggleDeleteSecretModal: () => void;
+}
+
+interface IRootState {
+  app: {
+    enrollSecret: IEnrollSecret[];
+  };
+}
+
+const baseClass = "delete-secret-modal";
+
+const DeleteSecretModal = ({
+  selectedTeam,
+  teams,
+  onDeleteSecret,
+  toggleDeleteSecretModal,
+}: IDeleteSecretModal): JSX.Element => {
+  const globalSecret = useSelector(
```

Comment:
I think this can be deleted since you aren't really using the value in `renderTeam`